### ISSUE TITLE
Multiple attendants issue

### ISSUE THREAD

**Issue by luc456** [alextselegidis/easyappointments#783]
Number of attendants is wrong when there are multiple services with multiple attendants at same time.

**Comment by luc456** [comment]
Here is the function fixed, the cause is order of "AND" and "OR" in where clausole.

```
    public function get_attendants_number_for_period(DateTime $slot_start, DateTime $slot_end, $service_id)
    {
        return (int)$this->db
            ->select('count(*) AS attendants_number')
            ->from('appointments')
            ->group_start()
            ->where('start_datetime <=', $slot_start->format('Y-m-d H:i:s'))
            ->where('end_datetime >', $slot_start->format('Y-m-d H:i:s'))
            ->where('id_services', $service_id) // ADDED
            ->group_end()
            ->or_group_start()
            ->where('start_datetime <', $slot_end->format('Y-m-d H:i:s'))
            ->where('end_datetime >=', $slot_end->format('Y-m-d H:i:s'))
            ->where('id_services', $service_id)  // ADDED
            ->group_end()
//            ->where('id_services', $service_id)  REMOVED
            ->get()
            ->row()
            ->attendants_number;
    }
```

**Comment by makayabou** [comment]
Is it in master or develop branch?

**Comment by luc456** [comment]
It's in both branches

**Comment by popod** [comment]
Seems to be fixed by this commit e857d3a37e62a33f9e048a8b47bea17852147479 in `develop` branch ?

**Comment by luc456** [comment]
it's correct!
